index.html

```diff
@@ -10,6 +90,7 @@
         <link rel="icon" href="./images/favicon.png" />
     </head>
     <body>
```

Comment:
Duplikuota visa `<head>` sekcija, hero ir headeris! atsargiai su copy paste 😄

Reply:
Lukai, o kaip taip galejo nutikti? Vienintelis dalykas ka padariau, tai po header dalies sumerginimo i developa, parsisiunciau nauja versija i kompa ir tiek, ar cia ir buvo beda tame? Jeigu taip, tai kaip tada atsinaujinti, kas jau yra padaryta? ir kitas klausimas, ka dabar man padaryti, kad sitaip nebutu? Nes pas mane 2 header daliu nera. ;/

Reply:
Neįsivaizduoju. Gali būt ir kad GIT'as sugrybavo, nėra jis visagalis.
Tai manau atsidaryti šitą branchą ir ištrinti ko nereikia.